apps/fabric-website/src/components/Nav/Nav.tsx

```diff
@@ -24,9 +25,11 @@ export class Nav extends React.Component<INavProps, INavState> {
       : null;
 
     return (
-      <nav className={ styles.nav } role='navigation'>
-        { links }
-      </nav>
+      <FocusZone ref='focusZone' direction={ FocusZoneDirection.vertical } role='menubar' >
```

Comment:
@micahgodbolt I am having trouble understanding the implementation here. We don't really need a role (menubar or menu) on focuszone. check https://www.w3.org/TR/wai-aria-1.1/#navigation and our Nav already follows this guidelines.